tools/issue-labeler/src/IssueLabelerService/TraigeRag.cs

```diff
@@ -22,6 +22,10 @@ public class TriageRag
         private static SearchIndexClient s_searchIndexClient;
         private ILogger<TriageRag> _logger;
 
+        private const int MinSubqueries = 3;
```

Comment:
nit:  Can we move these to the first class members?   Generally, C# conventions call for ordering by scope/visibility:

- const (which is static)
- static readonly
- static

and each type by visibility

- private
- protected
- internal
- public